controllers/cloud.redhat.com/config/types.go

```diff
@@ -0,0 +1,111 @@
+// Code generated by github.com/atombender/go-jsonschema, DO NOT EDIT.
+
+package config
+
+// application deployment configuration for cloud.redhat.com applications
+type AppConfig struct {
+	// Database corresponds to the JSON schema field "database".
```

Comment:
All these comments are unnecessary and annoying

Reply:
They are generated, is it really a showstopper, or are you giving me a hot take? :)

Reply:
yeah it's just a hot take.